pkg/v1/remote/multi_write.go

```diff
@@ -99,7 +99,7 @@ func MultiWrite(m map[name.Reference]Taggable, options ...Option) (rerr error) {
 	// Collect the total size of blobs and manifests we're about to write.
 	if o.updates != nil {
 		defer close(o.updates)
-		defer func() { sendError(o.updates, rerr) }()
+		defer func() { _ = sendError(o.updates, rerr) }()
```

Comment:
I'm curious if this complains with:
```suggestion
		defer sendError(o.updates, rerr)
```

Reply:
I gave up and disabled `errcheck` entirely when I saw how much more drudgery it would take to address them all. This is at least a signal to readers that the error is unchecked.